src/dpg_slam/dpg_slam.h

```diff
@@ -44,14 +45,38 @@ namespace dpg_slam {
      */
     class occupancyGrid {
     public:
+    
+    // Constructor without any nodes 
+    occupancyGrid(const DpgParameters &dpg_parameters, const PoseGraphParameters &pose_graph_parameters) :
+	      dpg_parameters_(dpg_parameters), pose_graph_parameters_(pose_graph_parameters) {}	    
```

Comment:
I think you still need to set the booleans for which points to include in this constructor.

Reply:
yes.